examples/ngrok-labeled.js

```diff
@@ -18,18 +18,18 @@ console.log("Node.js web server at", UNIX_SOCKET, "is running..");
 // setup ngrok
 const ngrok = require("@ngrok/ngrok");
 ngrok.consoleLog("INFO");
-builder = new ngrok.NgrokSessionBuilder();
+builder = new ngrok.SessionBuilder();
 builder.authtokenFromEnv().metadata("Online in One Line");
 
 builder.connect().then((session) => {
   session
-    .labeledTunnel()
+    .labeledListener()
```

Comment:
Maybe also renamed this to `edgeListener` 🤷

Reply:
Hrm, maybe both? `labeledEdgeListener`? There are already problems with discovery around labels: https://github.com/ngrok/ngrok-python/issues/66

I think I'd lean towards incorporating this into the other naming conversations going on, so we can roll out consistently if we decide to go that route.